services/ui-src/src/forms/mcpar/aap/aap.json

```diff
@@ -2,17 +2,20 @@
   "path": "/mcpar/program-information/add-plans",
   "intro": {
     "section": "Section A: Program Information",
-    "subsection": "Add Plans"
+    "subsection": "Add plans (A.7)",
```

Comment:
If I'm not mistaken the P should be capitalized here

Reply:
Got confirmation from David to use lowercase, like we've been doing for other pages